### ISSUE TITLE
A simple addition to the solo custom game 

### ISSUE THREAD

**Issue by Ruslan4ic** [tetrio/issues#985]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

Sometimes, we need a score requierment, because we sometimes want a score rush on it's own and share it to your friends, so that you will know who is the most score-efficient in your circle

### Describe the solution you'd like

Add the score option to the objective selection

### Describe alternatives you've considered

none for now

**Comment by ZaptorZap** [comment]
Oh, like back in TETR.IO's [5,000,000 BLAST](https://tetris.wiki/TETR.IO#5.2C000.2C000_Blast) mode—just without all of the weird score multiplication? I'll admit, this request sent me on a rollercoaster of responses since I mixed up my CUSTOM game "keys" and "objectives". (from "try CUSTOM games—150 line marathons are the default" to "BLITZ is already implemented if you wanted a score-based gamemode". Arguably, BLITZ _is still_ what you're looking for to compare who the most score-efficient player in your circle is..., but I digress)

**Comment by Ruslan4ic** [comment]
Yes, i want something similar to 5 million blast, but ajustable like to 10k, 100k, maybe a billion!
